config/keys.go

```diff
@@ -65,5 +194,74 @@ func LoadSigningKeys() (*SigningKeys, error) {
 		}
 		return nil, err
 	}
+
+	if err := validateKeys(&config); err != nil {
+		return nil, err
+	}
+
 	return &config, nil
 }
+
+// LoadExecSaveSigningKeys loads signing key, executes given function and then saves the signing key
+func LoadExecSaveSigningKeys(fn func(keys *SigningKeys) error) error {
```

Comment:
Will `UpdateSigningKeys` a better API name for this utility function?

Reply:
We already have methods called UpdateDefualt, Remove which updates the `SigningKeys` struct but actully doesnt saves them on disk, IMO its better to  have `LoadExecSaveSigningKeys` which shows what function is doing.